.github/workflows/generate-token-lists.yml

```diff
@@ -170,6 +170,12 @@ jobs:
             version: true
             command: yarn arbify --l2NetworkID 4078 --ignorePreviousList --tokenList https://tokenlist.arbitrum.io/ArbTokenLists/arbed_uniswap_labs.json --newArbifiedList ./src/ArbTokenLists/4078_arbed_uniswap_labs.json
```

Comment:
why does it still have the `--ignorePreviousList` flag?

Reply:
It wasn't based on the last master, fixed